### PR TITLE
User action required: TOS updates

### PR DESCRIPTION
**Pull request by goldoraf**

### CODE REVIEW

**Comment by goldoraf** [comment]
[(3) Gérer le changement de CGU dans les apps mobiles](https://trello.com/c/P6R3dPYu/970-3-g%C3%A9rer-le-changement-de-cgu-dans-les-apps-mobiles)

**Comment by goldoraf** [comment]
[(3) Gérer le changement de CGU dans les apps mobiles](https://trello.com/c/P6R3dPYu/970-3-g%C3%A9rer-le-changement-de-cgu-dans-les-apps-mobiles)

File: src/drive/mobile/containers/StackErrorBoundary.jsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    try {
+      await client.getOrCreateStackClient().fetch('GET', '/apps')
+      const wasBlocked = this.state.warnings.length !== 0
+      if (wasBlocked) {
+        this.setState({ warnings: [] })
+        router.push({ pathname: '/' })
+      }
+    } catch (e) {
+      if (e.status === 402) {
+        this.setState({ warnings: e.reason })
```
Comment (y-lohse) [inline src/drive/mobile/containers/StackErrorBoundary.jsx]: Is `e.reason` an array here? This isn't very clear.

File: src/drive/mobile/containers/StackErrorBoundary.jsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        size="small"
+        extension="full"
+        label={t('TOS.updated.disconnect')}
+        onClick={onRefuse}
+      />
+    </Modal.ModalDescription>
+  </Modal>
+))
+
+class StackErrorBoundary extends Component {
```
Comment (y-lohse) [inline src/drive/mobile/containers/StackErrorBoundary.jsx]: What do you think about renaming this component? In the end, it's not acting as a boundary at all, so maybe the name is misleading.

File: src/drive/mobile/containers/StackErrorBoundary.jsx (hunk 1 above)
Comment (goldoraf) [reply src/drive/mobile/containers/StackErrorBoundary.jsx]: yep, the stack returns an array in its response, that gets put into the `FetchError` `reason` property by `cozy-client`.

**Pull request merged**